packages/thirdweb/src/index.ts

```diff
@@ -93,3 +93,14 @@ export {
   parseGwei,
   parseUnits,
 } from "./utils/units.js";
+
+// pay
+export {
+  getRoute,
+  getSwapStatus,
+  sendSwap,
+  type SwapRoute,
+  type SwapRouteParams,
+  type SwapStatus,
+  type SwapStatusParams,
+} from "./pay/index.js";
```

Comment:
don't re-export from barrel files, instead export directly from the sources (see above other cases)